xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/saverights.vm

```diff
@@ -28,8 +28,15 @@
 
 #if ($!{services.csrf.isTokenValid("$!{request.getParameter('form_token')}")})
 #foreach($obj in $doc.getObjects($clsname)) ## XWiki.XWikiGlobalRights or XWiki.XWikiRights
+  ## In subwikis, users or groups may be referenced by "wiki:Space.Username" or "Space.Username"
+  #set ($wikiname = $xcontext.database)
```

Comment:
Please use camel-case as per our [Java code style](https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/).

Reply:
Well noted thanks. In the latest commit, there is no need to retrieve the wiki name anymore.